src/python/pants/backend/byolinter/register.py

```diff
@@ -0,0 +1,55 @@
+# This is convenient to have here
+# when actively developing on the functionality
+# but is actually intended to be written by the end-user consuming the lib
```

Comment:
this file is just here as a utility, it was easier for me to iterate this way, but the example actually relies on its in-repo version of this 'register.py': https://github.com/gauthamnair/example-pants-byo-tool/blob/main/pants-plugins/byo_linters/register.py